website/src/components/home/CanvasList.jsx

```diff
@@ -64,6 +66,11 @@ class CanvasList extends React.Component {
     if (!this.listenersAttached && nextProps.userId) {
       this.collectAndListenForCanvases(nextProps.userId);
     }
+    if (!userCanvasList) {
+      firebase.database().ref(`/users/${this.props.userId}/${RC.CANVASES}`).once('value').then((canvasListSnap) => {
```

Comment:
this information is held in the redux store under the canvases field. Because you loop through the redux canvases below on line 154, if a canvas has been removed from the redux store, it won't show up on the front page